packages/sui-test-contract/test/server/setupSpec.js

```diff
@@ -84,6 +85,16 @@ setupContractTests({
         ),
         response: gardenResponse,
         addMatchingRules: true
+      },
+      {
+        endpoint: '/apples/search/garden',
+        description: 'A request for getting a garden that fails',
+        state: 'I have not garden',
```

Comment:
```suggestion
        state: 'I don't have a garden',
```

or

```suggestion
        state: 'I have no garden',
```

sorry about the grammar comment 🤓